packages/docs/src/routes/docs/(qwikcity)/action/index.mdx

```diff
@@ -216,6 +216,11 @@ export const useAddUser = routeAction$(
     firstName: z.string(),
     lastName: z.string(),
   })
+  // Or, with Valibot:
```

Comment:
These comments will become a liability over time as we change things and we can't verify that they actually work.

Can we change them to an actual working example? (I know I am creating more work, please?) See https://github.com/BuilderIO/qwik/blob/main/packages/docs/src/routes/docs/integrations/modular-forms/index.mdx#L165-L231 for how it can be done

1. Create a working example here: https://github.com/BuilderIO/qwik/blob/main/packages/docs/src/routes/demo/integration/modular-forms/index.tsx
2. insert `<CodeSandbox src="/src/routes/demo/integration/modular-forms/index.tsx">`
3. Run `codesandbox.sync` in `packages/docs` to sync the `CodeSandbox` with the actual example.

Reply:
@mhevery 

I added CodeSandbox demos for the simple and advanced Valibot sections. Do we need CodeSandboxes for the smaller third and fourth sections?